extras/basic_example/public/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <title>Licode Basic Example</title>
-    <script type="text/javascript" src="erizo.js"></script>
+    <script type="text/javascript" src="http://192.168.33.99:8080/erizo.js"></script>
```

Comment:
I feel like I should know what IP address this is . . . but I don't.  I know it is a private network address.  So, not knowing what this IP is, is it okay that it is hard-coded?